pages/index.js

```diff
@@ -50,62 +56,159 @@ export const getServerSideProps = async () => {
   };
 };
 
-function HomePage({
+const getStateData = async (state) => {
+  const shootingsByCounty = await universalAPIFetch(
+    `count/shootings/statecounty/${state}`
+  );
+  const shootingsOverTime = await universalAPIFetch(
+    `count/shootings/overtime/${state}`
+  );
+  const last20Items = await universalAPIFetch(`shootings/last20/${state}`);
+  const topPoliceDepartments = await universalAPIFetch(
+    `count/shootings/pd/${state}`
+  );
+
+  return {
+    shootingsByCounty,
+    shootingsOverTime,
+    topPoliceDepartments,
+    last20Items,
+  };
+};
+
+export const getServerSideProps = async () => {
+  const nationalData = await getNationalData();
+  return nationalData;
+};
+
+function Dashboard({
   shootingsByState,
-  shootingsPerCapita,
+  shootingsByGeo,
   shootingsOverTime,
   topPoliceDepartments,
   last20Items,
+  selectedState,
+  setSelectedState,
+  setIsPerCapita,
 }) {
-  const [isPerCapita, setIsPerCapita] = useState(false);
-
   return (
     <Container>
       <Head />
+      <Row>
+        <Col sm="auto">
+          <h2>Police Killings in 2020</h2>
```

Comment:
Great to see a main heading for the page, but would recommend using an `h1` element here- I don't think the page has one yet, and assistive tech like screen readers look for one to pass along the context of the page.